requirements.txt

```diff
@@ -30,3 +30,4 @@ qrcode[pil]~=6.1
 requests~=2.31.0
 rlp==1.2.0
 unflatten~=0.1
+git+https://github.com/danielfett/sd-jwt.git@main
```

Comment:
I believe this has changed to be in openwallet foundation's repos; might be a good idea to update here.